### PR TITLE
backport 2022.02.xx - #8529 maxItems is now used from Identify plugin (#8539)

### PR DESCRIPTION
**Pull request by MV88**
## Description

backport 2022.02.xx - #8529 maxItems is now used from Identify plugin (#8539)

### CODE REVIEW

**Review by ale-cristofori — APPROVED** [review]
All ok by for me

**Comment by tdipisa** [comment]
@MV88 @ale-cristofori please wait for the release 2022.02.00 before merging this. Thank you.

**Pull request merged**
